00new_packages.autobuild

```diff
@@ -13,5 +13,5 @@ end
 
 cmake_package 'gui/rock_webapp'
 ruby_package 'tools/rest_api' do |pkg|
-    Autoproj.env_set 'ROCK_WEBAPP', File.join(pkg.prefix, 'share', 'rest_api')
+    Autoproj.env_set 'ROCK_WEBAPP', File.join(Autoproj.build_dir, 'share', 'rest_api')
```

Comment:
* ROCK_WEBAPP is a single path solely used as configuration directory for the rock-rest-api program, this is a singeton by definition, as there cannot be two configurations for a single instance of a program.
* the envvar WAS already pkg.prefix, but with autoproj 2 this points to the SOURCE directory for ruby packages, not the installation directory (that is why this PR was created)

Apart from the different behavior between autoproj 1 and 2 concerning pkg.prefix for ruby packages, we could indeed think about adding the mentioned ROCK_WEBAPP_PLUGIN_PATH, which then, should be a real path with multiple entries.

But adding that is a new feature to the REST API, while this PR is a simple bugfix (or workaround to a autoproj bug).